commands/util/pvp-rank.js

```diff
@@ -0,0 +1,161 @@
+    .setThumbnail(args['pvp_pokemon'].imageURL)
+    .setFooter(`${message.author.username} || Data retrieved from https://gostadium.club`);
+
+    let userCommand = `${message.client.commandPrefix}${command}`; //Grabs the !great, !ultra or w/e from user inputs.
+    let userPokemon = `${args['pvp_pokemon'].pbName.titleCase()}`; //Grabs the accepted pokemon name.
+    let userIVString = `${this.inputATKiv} ${this.inputDEFiv} ${this.inputSTAiv}`; //Grabs the accepted IV sets (0-15 for ATK,DEF,STA)
+    let responseCommand = `${message.author}, here are your \`${userCommand} ${userPokemon} ${userIVString}\` results:`; //Combined the above into the whole accepted command.
+    let response = await message.channel.send(responseCommand, embed);
+    response.preserve = true;
+  }
+}
+  class RankCommand extends Commando.Command {
+    constructor(client) {
+    super(client, {
+      name: 'rank',
+      group: CommandGroup.UTIL,
+      memberName: 'rank',
+      aliases: [/*'pvp-rank', 'pvp-iv', */'great', 'ultra'], //Aliases are other user commands that call this command file
+      description: 'Provides PvP data based on a pokemon species\'s IVs, including rank and CP.', //Help info
+      details: 'Use this command in #rank-checker to obtain information on the PvP ranking of a specific IV combination for a specific species of pokemon.' +
+          '\n!great - This command restricts results to **Great League**\n!ultra - This command restricts results to **Ultra League**', //Help info
+      examples: ['!<league> <pokemon> <Attack IV> <Defense IV> <Stamina IV>\n!great altaria 4 1 2\n!ultra blastoise 10 14 15\n'], //Help info
+      guarded: false, //????
+      args: [
+        {
+          key: 'pvp_pokemon', //Variable pokemon is the first entry in the command
+          prompt: 'Which species of pokemon would you like to evaluate a specific IV combination PvP rank for?',
+          type: 'counterpokemontype', //Ensures that the entered information is a pokemon name.
+        },
+        {
+          key: 'ATK_IV',
+          prompt: 'Attack IV?',
+          type: 'integer',
+          min: 0,
+          max: 15,
+        },
+        {
+          key: 'DEF_IV',
+          prompt: 'Defense IV?',
+          type: 'integer',
+          min: 0,
+          max: 15,
+        },
+        {
+          key: 'STA_IV',
+          prompt: 'Stamina IV?',
+          type: 'integer',
+          min: 0,
+          max: 15, //Can we add additional args for optional additional parameters for specific searches? For example, minimum IV
+        },],
+      argsPromptLimit: 3, //What is this?
+    });
+  }
+
+  async run(message, args) {
+
+    String.prototype.titleCase = function () {
+      return this.replace(/\w\S*/g, function(str) {return str.charAt(0).toUpperCase() + str.substr(1).toLowerCase();});
+    };
+
+    let userCommand = message.content.match(RegExp(`^\\s*${message.client.commandPrefix}\\s*(\\S+)\\s*`))[1].toLowerCase();

```

Comment:
To avoid RegEx, you can simply use standard string functions as the command will _always_ be first in content:
`message.content.split(' ').shift().substring(1)`

Reply:
I will test this out, thank you!

Reply:
Looking back at this, I forgot `! Great` works as well because of how Commando works. You can add `.substring(1).trim()` before .split() and remove the last sub string call.